src/components/RepositoryReleasesComparator.tsx

```diff
@@ -19,12 +19,12 @@ const RepositoryReleasesComparator = () => {
   const { repository, fromVersion, toVersion } = useComparatorState()
 
   return (
-    <>
+    <Flex direction="column" height="full">
       <Container variant="fluid">
         <RepositoriesComparatorFilters />
       </Container>
       <Divider mt={4} />
-      <Box bgColor="secondaryBg" pt={2} minHeight="full">
+      <Box bgColor="background3" pt={2} flex="1 0 auto">
```

Comment:
This was causing the comparator to exceed the footer and take more height than expected. It's fixed now.